<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingDlg</class>
 <widget class="QDialog" name="CSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_topbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_serverIP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>serverIP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_serverother1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>other1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_serverother2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>other2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_port">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_serverPort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>serverPort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_confirm">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>402</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_test">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>402</y>
     <width>66</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>link</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>402</y>
     <width>102</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle(&amp;C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
